mscore/seq.cpp

```diff
@@ -562,6 +562,14 @@ void Seq::processMessages()
                         break;
                   case SeqMsgId::PLAY:
                         putEvent(msg.event);
+                        // Update initList immediately if there're some changes in mixer
```

Comment:
I'm not sure I get the purpose of this new block. Could we do this in the ui thread?